modules/cluster-logging-updating-logging.adoc

```diff
@@ -69,33 +65,12 @@ The Cluster Logging Operator is shown as 4.5. For example:
 +
 ----
 Cluster Logging
-4.5.0-202006160135 provided
+4.5.0-202005201915 provided
 by Red Hat, Inc
 ----
 
 . Check the logging components:
 
-.. Ensure that the Elasticsearch Pods are using a 4.5 image:
-+
```

Comment:
As it is not easy to get an image version,  I suggest deleting the paragraphs ‘Ensure xxxx are using a 4.5 image‘.  Instead, add the following verification steps.
1) Verify logStore is upgraded to 4.5
     oc exec -c elasticsearch $es-pod -- indcies. The output include indices 'app-0000x, infra-0000x, audit-0000x, .security'
oc exec -c elasticsearch elasticsearch-cdm-tutz1ydx-1-5774855ffc-f8s4p -- indices
Tue Jun 30 14:30:54 UTC 2020
health status index                                                                 uuid                   pri rep docs.count docs.deleted store.size pri.store.size
green  open   infra-000008                                                          bnBvUFEXTWi92z3zWAzieQ   3   1     222195            0        289            144
green  open   infra-000004                                                          rtDSzoqsSl6saisSK7Au1Q   3   1     226717            0        297            148
green  open   infra-000012                                                          RSf_kUwDSR2xEuKRZMPqZQ   3   1     227623            0        295            147
green  open   .kibana_7                                                             1SJdCqlZTPWlIAaOUd78yg   1   1          4            0          0              0
green  open   .operations.2020.06.30                                                aOHMYOa3S_69NJFh2t3yrQ   3   1    4206118            0       8998           4499
green  open   project.local-storage.d5c8a3d6-30a3-4512-96df-67c537540072.2020.06.30 O_Uldg2wS5K_L6FyqWxOZg   3   1      91052            0        135             67
green  open   infra-000010                                                          iXwL3bnqTuGEABbUDa6OVw   3   1     248368            0        317            158
green  open   .searchguard                                                          rQhAbWuLQ9iuTsZeHi_2ew   1   1          5           64          0              0
green  open   infra-000009                                                          YN9EsULWSNaxWeeNvOs0RA   3   1     258799            0        337            168
green  open   infra-000014                                                          YP0U6R7FQ_GVQVQZ6Yh9Ig   3   1     223788            0        292            146
green  open   infra-000015                                                          JRBbAbEmSMqK5X40df9HbQ   3   1     224371            0        291            145
green  open   .orphaned.2020.06.30                                                  n_xQC2dWQzConkvQqei3YA   3   1          9            0          0              0
green  open   infra-000007                                                          llkkAVSzSOmosWTSAJM_hg   3   1     228584            0        296            148
green  open   infra-000005                                                          d9BoGQdiQASsS3BBFm2iRA   3   1     227987            0        297            148
green  open   .kibana.647a750f1787408bf50088234ec0edd5a6a9b2ac                      _l911Z8dSI23py6GDtyJrA   1   1          5            4          0              0
green  open   project.ui-upgrade.29cb9680-864d-43b2-a6cf-134c837d6f0c.2020.06.30    5A_YdRlAT3m1Z-vbqBuGWA   3   1         24            0          0              0
green  open   infra-000003                                                          1-goREK1QUKlQPAIVkWVaQ   3   1     226719            0        295            147
green  open   .security                                                             zeT65uOuRTKZMjg_bbUc1g   1   1          5            0          0              0
green  open   .kibana_-377444158_kubeadmin                                          wvMhDwJkR-mRZQO84K0gUQ   3   1          1            0          0              0
green  open   infra-000006                                                          5H-KBSXGQKiO7hdapDE23g   3   1     226676            0        295            147
green  open   project.nw-test.6233ad57-aff0-4d5a-976f-370636f47b11.2020.06.30       dtc6J-nLSCC59EygeV41RQ   3   1         10            0          0              0
green  open   infra-000001                                                          eH53BQ-bSxSWR5xYZB6lVg   3   1     341800            0        443            220
green  open   .kibana-6                                                             RVp7TemSSemGJcsSUmuf3A   1   1          4            0          0              0
green  open   infra-000011                                                          J7XWBauWSTe0jnzX02fU6A   3   1     226100            0        293            146
green  open   app-000001                                                            axSAFfONQDmKwatkjPXdtw   3   1     103186            0        126             57
green  open   infra-000016                                                          m9c1iRLtStWSF1GopaRyCg   3   1      13685            0         19              9
green  open   infra-000002                                                          Hz6WvINtTvKcQzw-ewmbYg   3   1     228994            0        296            148
green  open   project.qitang.2c05acbd-bc12-4275-91ab-84d180b53505.2020.06.30        MUm3eFJjSPKQOJWoHskKqw   3   1      12262            0         14              7
green  open   infra-000013                                                          KR9mMFUpQl-jraYtanyIGw   3   1     228166            0        298            148
green  open   audit-000001                                                          eERqLdLmQOiQDFES1LBATQ   3   1          0            0          0              0
2) Verify the collection is updated to 4.5 (There is a initContainers)
oc get ds fluentd -o json |jq '.spec.template.spec.initContainers[0].name'
"fluentd-init"
3) Verify log visualization are updated to 4.5( There is a new CRD kibana. and the pod in status are running)
     oc get kibana kibana -o json |jq '.status'
[
  {
    "clusterCondition": {
      "kibana-5fdd766ffd-nb2jj": [
        {
          "lastTransitionTime": "2020-06-30T14:11:07Z",
          "reason": "ContainerCreating",
          "status": "True",
          "type": ""
        },
        {
          "lastTransitionTime": "2020-06-30T14:11:07Z",
          "reason": "ContainerCreating",
          "status": "True",
          "type": ""
        }
      ]
    },
    "deployment": "kibana",
    "pods": {
      "failed": [],
      "notReady": [
        "kibana-5fdd766ffd-nb2jj"
      ],
      "ready": []
    },
    "replicaSets": [
      "kibana-5fdd766ffd"
    ],
    "replicas": 1
  }
]

4) Verify the curation is updated to 4.5. (There are elasticsearch-delete and elasticsearch-rollover index )
#oc get cronjob -o name
cronjob.batch/curator
cronjob.batch/elasticsearch-delete-app
cronjob.batch/elasticsearch-delete-audit
cronjob.batch/elasticsearch-delete-infra
cronjob.batch/elasticsearch-rollover-app
cronjob.batch/elasticsearch-rollover-audit
cronjob.batch/elasticsearch-rollover-infra

Reply:
@anpingli Thanks for these outputs. However, we have been told not to use `jq` in the docs [1]. Is there another way to get this information? 
The user could grep for the `fluentd-init` container. I did the following in 4.4, I assume the same is true in 4.5 to get `"containerName": "fluentd-init"`
```
$ oc get ds fluentd -o json | grep fluentd
"containerName": "fluentd"

```
The user could view the full output for the Kibana CRD. 
Is your command correct? `oc get kibana kibana` not `oc get customresourcedefinition kibana`
 
[1] https://bugzilla.redhat.com/show_bug.cgi?id=1764726#c6

Reply:
Yes, you can use the grep instead of jq.      jq is the best command and had been released in OCP repository     http://download.eng.bos.redhat.com/rcm-guest/puddles/RHAOS/AtomicOpenShift-signed/4.5/latest/x86_64/os/Packages/

Reply:
Yes, it is 'oc get kibana kibana'.   you can use 'oc get kibana kibana -o json'

Reply:
Got it. Thanks. Made the updates.